core/eolearn/tests/test_eodata_io.py

```diff
@@ -565,3 +565,53 @@ def test_partial_loading_fails(fs_loader: type[FS], eopatch: EOPatch, use_zarr:
 
         with pytest.raises(IndexError):
             EOPatch.load(path="patch-folder", filesystem=temp_fs, temporal_selection=temporal_selection)
+
+
+@mock_s3
+@pytest.mark.parametrize("fs_loader", FS_LOADERS)
+@pytest.mark.parametrize("use_zarr", [True])
```

Comment:
removed the fs_loader parametrization as well, since only TempFS works with it.
Though maybe i should've left it in if they fix it in the future?

The pass function is still called because it has to disable the test if the dependency is not present

Reply:
>  Though maybe i should've left it in if they fix it in the future?

Not sure about this. I guess then the same argument goes for other similar tests? Then I'd say it's better to be consistent

Reply:
other similar tests also have the `use_zarr` flag, but partial saving is zarr specific